plugins/trace_api_plugin/trace_api_plugin.cpp

```diff
@@ -285,6 +313,11 @@ struct trace_api_rpc_plugin_impl : public std::enable_shared_from_this<trace_api
             http_plugin::handle_exception("trace_api", "get_block", body, cb);
          }
       });
+
+      auto ro_api = app().get_plugin<trace_api_plugin>().get_read_only_api();
+      app().get_plugin<http_plugin>().add_api({
+         CHAIN_API_RO_CALL(get_transaction, http_params_types::params_required),
```

Comment:
I thought we were going to name this `get_transaction_trace`

Reply:
Fixed.